VP-EN :: New Features

osulli opened the pull request.

Commit:
Teleport locations

Added Tarot Card / graffiti locations
Added 3 new bonus locations
Added Sandra Dorsett (NPC) location

Commit:
Lots of new protocols

Added some new protocols.. Such as:
Infinite Stamina
Infinite Oxygen
Rocket Boots / Slow Fall
Forced NPC Death
Slow Motion
Discover all '?' marks on map

Cumulative stat adding (+ max health, + max armor, etc)

New teleport locations

Commit:
Merge remote-tracking branch 'upstream/main' into new-features-v2

Commit:
New recipes / missing recipes

Added missing / new recipes

Ozobs nose recipe changed

Commit:
Fixing small mistakes

Formatting
Fixed the player stat modification stuff
Fixed a typo here and there

Commit:
Formatting

osulli commented on mods/braindance_protocol/cheats/teleport.lua:
```diff
@@ -48,14 +48,14 @@ function Teleport.GenericTotentanzClub()
 end
 
-function Teleport.GenericDMCInside()
+function Teleport.GenericDeMCInside()
```
Is this a typo? Why `DeMC` instead of `DMC` if it is for `Dark Matter Club` ?

VP-EN replied:
Oh, whoops. Should've been DaMC or DMC. I'll fix that 👍

osulli commented on mods/braindance_protocol/cheats/teleport.lua:
```diff
@@ -48,14 +48,14 @@ function Teleport.GenericTotentanzClub()
 end
 
-function Teleport.GenericDMCOutside()
+function Teleport.GenericDeMCOutside()
```
Is this a typo? Why DeMC instead of DMC if it is for Dark Matter Club ?

VP-EN replied:
Oh, whoops. Should've been DaMC or DMC. I'll fix that 👍

osulli commented on mods/braindance_protocol/cheats/teleport.lua:
```diff
@@ -153,6 +153,13 @@ function Teleport.GenericTheAtlantis()
 end
 
+function Teleport.GenericNomadStartLocationAWACSZone()
+    local moduleName = "Teleporting to Nomad Starting Location - Awacs Zone"
```
Can this be changed to `AWACS`

VP-EN replied:
Yes, will do

VP-EN replied:
Oh wait a minute, do you mean the name of the function or the actual location?

osulli reviewed (commented):
LGTM @VP-EN

Commit:
New locations + fixes

Added a few new locations
Fixed a few typos

Commit:
Fixed teleport commands

Fixed teleport commands. Apparently overlooked the quotation marks.
Also forgot to give names to new teleporting locations

Commit:
Merge branch 'main' into new-features-v2
